src/clients/sidecar/models/KafkaClusterConfigCredentials.ts

```diff
@@ -26,13 +26,20 @@ import {
   BasicCredentialsFromJSONTyped,
   BasicCredentialsToJSON,
 } from "./BasicCredentials";
+import type { OAuthCredentials } from "./OAuthCredentials";
+import {
+  instanceOfOAuthCredentials,
+  OAuthCredentialsFromJSON,
+  OAuthCredentialsFromJSONTyped,
+  OAuthCredentialsToJSON,
+} from "./OAuthCredentials";
 
 /**
  * @type KafkaClusterConfigCredentials
  * The credentials for the Kafka cluster, or null if no authentication is required
  * @export
  */
-export type KafkaClusterConfigCredentials = ApiKeyAndSecret | BasicCredentials;
+export type KafkaClusterConfigCredentials = ApiKeyAndSecret | BasicCredentials | OAuthCredentials;
```

Comment:
cc @noeldevelops

Reply:
OAuthCredentials is untested, let's hold off on adding support for it in the connection form.